docs/steamdeck-and-linux/README.md

```diff
@@ -0,0 +1,3 @@
+# SteamDeck and Linux
+
+Northstar is officially supported on SteamDeck and Linux. Check the subpages for installation, usage, and troubleshooting information.
```

Comment:
Probably also want to link to it directly ^^

Reply:
What is it you want there to be a direct link to? Each of the subpages? I followed the format of `installing-northstar/using-northstar/README.md`.

Reply:
Yeah, I'd just link all of them tbh.

(Sorry for late response, I thought I responded already, whoops)